pkg/set/string_set.go

```diff
@@ -10,6 +10,7 @@ type stringSet struct {
 func NewStringSet() *stringSet {
 	s := &stringSet{}
 	s.m = make(map[string]struct{})
+	s.v = []string{}
```

Comment:
`stringSet.ToSlice()` was returning `[]string(nil)` when the set was blank without this change.